packages/mui-base/src/Collapsible/Root/CollapsibleRoot.test.tsx

```diff
@@ -64,7 +64,13 @@ describe('<Collapsible.Root />', () => {
       expect(content).to.have.attribute('hidden');
     });
 
-    it('uncontrolled mode', async () => {
+    it('uncontrolled mode', async function test(t = {}) {
+      if (/jsdom/.test(window.navigator.userAgent)) {
+        // @ts-expect-error to support mocha and vitest
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+        this?.skip?.() || t?.skip();
+      }
```

Comment:
No way to make this cleaner?

Reply:
Not yet. We'll get rid of this once we drop mocha completely (= soon, I hope)